src/chronos/traveler/handler.ts

```diff
@@ -0,0 +1,18 @@
+import { Transfer } from '../../../generated/Traveler/ChronosTraveler'
+import { NftHandle } from '../../utils/nft-handle'
+
+const TYPE = 'TRAVELER'
+const PLATFORM = 'Chronos'
+const STATE = 'NONE'
```

Comment:
Maybe we should create a enum file to list all platforms, types and state (like schema) and only import it here?

Reply:
We going to remove these.